cpython/VITABUILD

```diff
@@ -0,0 +1,20 @@
+pkgname=cpython
+pkgver=2.7
+pkgrel=1
+url="https://github.com/SonicMastr/cpython-vita"
+source=("git://github.com/SonicMastr/cpython-vita.git")
+sha256sums=('SKIP')
+depends=('openssl libzip xz zlib bzip2')
+
+build() {
+  echo ""
+}
+
+package () {
+  cd cpython-vita
+  sh build_vita.sh $pkgdir
```

Comment:
`sh` could be `dash`

Reply:
Not every system have dash. But all have sh.

Reply:
that's a problem. every linux distro can have different default shell.
debian(includes ubuntu) uses dash for the default shell. and it isn't fully compatible as bash.

Reply:
It doesn't need to be bash-compatible, if build_vita doesn't use any bash extensions. dash is fully sh-compatible

Reply:
So as long as build_vita doesn't use any bash/dash/etc. extensions, sh is a safe default

Reply:
@frangarcj short answer. it's okay.

long answer. @isage 's last comment is a point. we can accept the patch until `build_vita.sh` is simple `sh` script.
but I prefer moving commands of script to `VITABUILD` directly. :)

Reply:
Yeah. I can see that for `build_vita.sh` at least. The main reason I went with using the script instead was because of the `install_extra_modules' step. Since I intend for that part of the build to be changed and added to, I just use scripts to prevent the need to constantly update the VITABUILD each time a new module was added. `build_vita.sh` could be moved entirely into the VITABUILD as well, but I'm also not sure what features could be enabled in the future that I could just change from the master branch instead. So I just opted to make them VITABUILD compatible by allowing it to change the install prefix as an argument.